source/aggregation/vector-search.txt

```diff
@@ -0,0 +1,213 @@
+.. _php-vector-search:
+
+===================
+Atlas Vector Search
+===================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: code example, semantic, text, embeddings
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to perform searches on your documents
+by using the Atlas Vector Search feature. The {+library-short+} allows you to
+perform Atlas Vector Search queries by using the :ref:`Aggregation Builder API
+<TODO DOCSP-43919>`.
+
+.. note:: Deployment Compatibility
+
+     You can use the Atlas Search feature only when
+     you connect to MongoDB Atlas clusters. This feature is not
+     available for self-managed deployments.
+
+To learn more about Atlas Vector Search, see the :atlas:`Overview
+</atlas-vector-search/vector-search-overview/>` in the
+Atlas documentation. The Atlas Vector Search implementation for the
+{+library-short+} internally uses the ``$vectorSearch`` aggregation operator
+to perform queries. To learn more about this operator, see the
+:atlas:`$vectorSearch
+</atlas-vector-search/vector-search-stage/#syntax>` reference in the
+Atlas documentation.
+
+.. note::
+
+   You might not be able to use the methods described in this guide for
+   every type of Atlas Vector Search query. For more complex use cases, you can
+   create standard aggregation pipelines. To learn more, see the
+   :ref:`php-aggregation` guide.
```

Comment:
It looks like you have took this statement from the [Laravel package](https://github.com/mongodb/laravel-mongodb/pull/3255/files#diff-46a82652f3f916c449934e197b5a991f5c0b86a88ade23c3d8acb32cc378cd31R45-R48). But for the PHPLIB, the aggregation builder can be used to do everything.

Reply:
This is the situation in which I asked for a way to describe the BSON document based agg pipeline syntax. If you're saying that there aren't any certain edge cases that might need that as an option, then I'll remove this sentence.

Reply:
I thought you were talking about Laravel.
You can indicate that the Builder is not the only way to write the search aggregation pipeline, and that it's possible to use simple arrays.

Reply:
I'll leave that content for https://jira.mongodb.org/browse/DOCSP-43919!